playground/main.js

```diff
@@ -66,12 +68,24 @@ const renderPlaygroundWidget = (options = {}) => {
     console.log('===== playground widget ready event received =====');
   });
 
-  signIn.on('afterRender', (context) => {
+  signIn.on('afterRender', (context, settings) => {
     // handle `afterRender` event.
     // use `console.log` in particular so that those logs can be retrived
     // in testcafe for assertion
     console.log('===== playground widget afterRender event received =====');
     console.log(JSON.stringify(context));
+
+    // assert english leaks if locale set to ok-PL
+    if (settings.get('language') === 'ok-PL') {
```

Comment:
make `ok-PL` a constant  ?

Reply:
I think this is fine as we are only checking for the language code in this one place.